src/components/DonationItemsTable/DonationItemsTable.jsx

```diff
@@ -12,76 +22,238 @@
+      }
+    };
+
+    const fetchNotifications = async () => {
+      const response = await backend.get('/notification/0');
+      setNotification(response.data);
+    }
+    checkIsAdmin();
+    fetchNotifications();
+  }, [backend]);
   return (
-    <div>
-      <Tabs>
-        <TabList>
-          <Tab>All</Tab>
-          <Tab>Food</Tab>
-          <Tab>Misc.</Tab>
-        </TabList>
-        <TabPanels>
-          <TabPanel>
-            <DonationItems category="all" />
-          </TabPanel>
-          <TabPanel>
-            <DonationItems category="Food" />
-          </TabPanel>
-          <TabPanel>
-            <DonationItems category="Misc." />
-          </TabPanel>
-        </TabPanels>
-      </Tabs>
-    </div>
+    <>
+      {isAdminUser && (
+        <div className={classes.container}>
+          <div className={classes.ditTitleContainer}>
+            <Text>Donation Items</Text>
+            <NotificationsDrawer notificationsData={notification} />
+          </div>
+          <Tabs marginTop="24px" isFitted="true" colorScheme="teal">
+            <div>
+              <TabList w="185px" mb={-10}>
+                <Tab>All</Tab>
+                <Tab>Food</Tab>
+                <Tab>Misc.</Tab>
+              </TabList>
+            </div>
+            <TabPanels >
+              <TabPanel padding="0">
+                <DonationItems category="all" />
+              </TabPanel>
+              <TabPanel padding="0">
+                <DonationItems category="Food" />
+              </TabPanel>
+              <TabPanel padding="0">
+                <DonationItems category="Misc." />
+              </TabPanel>
+            </TabPanels>
+          </Tabs>
+        </div>
+      )}
+    </>
   );
 };
 
 const DonationItems = ({ category }) => {
   const [data, setData] = useState([]);
   const { backend } = useBackend();
-  const TABLE_HEADERS = ['Id', 'Name', 'Quantity Type', 'Price', 'Category'];
+  const [currentItemNum, setCurrentItemNum] = useState(0);
+  const [currentPageNum, setCurrentPageNum] = useState(1);
+  const [pageLimit, setPageLimit] = useState(1);
+  const [selectedItem, setSelectedItem] = useState(null);
+  const [isEdit, setIsEdit] = useState(false);
+  const {
+    isOpen: deleteModalIsOpen,
+    onOpen: deleteModalOnOpen,
+    onClose: deleteModalOnClose,
+  } = useDisclosure();
+  const {
+    isOpen: donationsModalIsOpen,
+    onOpen: donationsModalOnOpen,
+    onClose: donationsModalOnClose,
+  } = useDisclosure();
+
+  const TABLE_HEADERS = ['Id', 'Name', 'Quantity Type', 'Price', 'Category', ''];
+
   useEffect(() => {
-    const getData = async () => {
-      try {
-        const response = await backend.get(`/value/filter/${category}`);
-        setData(response.data);
-      } catch (error) {
-        console.error('Error getting donation items:', error);
-        setData([]);
-      }
-    };
-    getData();
-  }, [backend, category]);
+    loadInfo();
+  }, [currentPageNum, category]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'loadInfo'. Either include it or remove the dependency array.